tests/functional/test_provider.py

```diff
@@ -79,6 +84,45 @@ def test_get_receipt_exists_with_timeout(eth_tester_provider, vyper_contract_ins
     assert receipt_from_provider.receiver == vyper_contract_instance.address
 
 
+def test_get_receipt_without_raising(eth_tester_provider, vyper_contract_instance, owner):
+    # The Contract raises empty revert when setting number to 5.
+    receipt_from_invoke = vyper_contract_instance.setNumber(
```

Comment:
This part of the test might make more sense in `test_contract_instance()`

`def test_invoke_failing_method_without_raising()`

Reply:
Can do, I'll take a look at cleaning up some test stuff in the upcoming PR for reverts since that will need to add in testing changes